### DISCUSSION TITLE
Dark theme solution for KDE users

### DISCUSSION THREAD

**Discussion by Elliria** [autokey/autokey#835] [General]
Hey there, it looks like there's a work-around that will ease the issue of dark themes making it difficult or impossible to read some of the text in an AutoKey window. There's a desktop effect called **Invert** that makes it so that you can invert the colors/darkness and lightness of any window at any time, so with a click, you'll be able to read and find your way around in a window that's suffering from theme misbehavior. See it in action here in [this YouTube video](https://youtu.be/ybWFSIWEfVM?t=480).

**Comment by josephj11** [comment]
I can't find invert in 18.04, but I just partially broke my notebook screen, so I have to figure out how to afford a new notebook - which will get 22.04 because I will have to do a clean install anyway.

That video is full of useful stuff that I didn't know, but it goes by too fast. I need a transcript/article. I will search for that later.

I am thinking about subscribing to his channel.

**Comment by Elliria** [comment]
Yeah, I was actually looking up something else and happened to come across the **Invert** Desktop Effect and figured it might "solve" the AutoKey issue with dark themes for those of you who use them. 

The rest of his stuff is equally as fascinating. I've already checked out several of his videos and they remind me that there's always something new to learn about KDE. It's too huge for any one person to know it all, so it's an everlasting adventure of discovery.

I'm sorry about your notebook screen. You might be able to get a replacement in the meanwhile. If you do get a new notebook, though, I'm sure you'll love it and will definitely enjoy some of the new perks in Kubuntu 22.04 LTS.
